Week1/Section1/Exercise4/Source.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+* Exercise 4 - Underflow and Overflow
+* Write a program that will illustrate overflow and underflow.
+*/
+
+#include <iostream>
+#include <climits>
+
+using namespace std;
+
+int main() {
+
+	int max = 128;
```

Comment:
This variable is not being used, is it? Unused code will make your overall code harder to understand, be mindful of removing stuff you no longer need :)